packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,417 @@
+            </StackLayout>
+          }
+          endItem={
+            <Button
+              onClick={handleClick}
+              variant="secondary"
+              aria-expanded={expandedDetails}
+              aria-label={"expand contact details"}
+            >
+              {expandedDetails ? (
+                <ChevronUpIcon aria-hidden />
+              ) : (
+                <ChevronDownIcon aria-hidden />
+              )}
+            </Button>
+          }
+        />
+        {expandedDetails && (
+          <StackLayout
+            direction={"row"}
+            gap={2}
+            style={{
+              borderTop: "solid",
+              borderWidth: "1px",
+              borderColor: "var(--salt-separable-primary-borderColor)",
+              padding: "var(--salt-spacing-100)",
+            }}
+          >
+            <StackLayout direction={"column"} gap={0.5}>
+              {basicContact.metadata.map((metadata) => {
+                return (
+                  <Text variant="secondary" key={metadata.label}>
+                    {metadata.label}
+                  </Text>
+                );
+              })}
+            </StackLayout>
+            <StackLayout direction={"column"} gap={0.5}>
+              {basicContact.metadata.map((metadata) => {
+                return <Text key={metadata.label}>{metadata.value}</Text>;
+              })}
+            </StackLayout>
+          </StackLayout>
+        )}
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+const contactList = [
+  {
+    primary: "Jane Doe",
+    sid: "O12",
+    avatarImage: persona,
+    number: "+442077431102",
+    email: "email@company.com",
+  },
+  {
+    primary: "Logan Rider",
+    sid: "U34",
+    avatarImage: persona2,
+    number: "+442077431102",
+    email: "email@company.com",
+  },
+  {
+    primary: "Paul Hill",
+    sid: "L56",
+    avatarImage: persona3,
+    number: "+442077431102",
+    email: "email@company.com",
+  },
+];
+
+const adornmentMap: Record<string, JSX.Element> = {
+  O12: <Avatar src={persona} size={1} />,
+  U34: <Avatar src={persona2} size={1} />,
+  L56: <Avatar src={persona3} size={1} />,
+};
+
+export const List = () => {
+  const [selectedContactId, setSelectedContactId] = useState<string[]>([
+    contactList[0].sid,
+  ]);
+
+  const handleSelectionChange: DropdownNextProps["onSelectionChange"] = (
+    event,
+    newSelected
+  ) => {
+    setSelectedContactId(newSelected);
+  };
+
+  const adornment: JSX.Element = adornmentMap[selectedContactId[0] ?? ""];
+
+  return (
+    <DropdownNext
+      style={{ width: "266px" }}
+      startAdornment={adornment}
+      onSelectionChange={handleSelectionChange}
+      defaultValue={contactList[0].primary}
+    >
+      {contactList.map((contact) => (
+        <Option
+          key={contact.primary}
+          value={contact.sid}
+          textValue={contact.primary}
+        >
+          <StackLayout
+            direction={"row"}
+            gap={1}
```

Comment:
in `list.png`, guidance says to use `gap={3}` 🤔

Reply:
the gap 3 is used in the Flow Layout on line 401, this gap is between the avatar and the rest of the content so should be spacing-100 ( gap 1)